cli/src/index.ts

```diff
@@ -4188,6 +4179,30 @@ function retryWithExponentialBackoff<T>(
     return attempt(0);
 }
 
+function buildVerifierArgs(chain: Chain): string[] {
+    const verifier = configuration.get(chain, "verifier", { reportError: true });
+    const apiKey = configuration.get(chain, "scan_api_key", { reportError: true });
+
+    if (!apiKey) {
+        process.exit(1);
+    }
+
+    const verifierType = verifier || "etherscan";
+
+    switch (verifierType.toLowerCase()) {
+        case "blockscout":
+            const verifierUrl = configuration.get(chain, "verifier_url", { reportError: true });
+            if (!verifierUrl) {
+                console.error(`verifier_url is required when using blockscout verifier for ${chain}`);
+                process.exit(1);
+            }
+            return ["--verify", "--verifier", "blockscout", "--verifier-url", verifierUrl, "--verifier-api-key", apiKey];
+        case "etherscan":
+        default:
+            return ["--verify", "--verifier", "etherscan", "--verifier-api-key", apiKey];
```

Comment:
is there a reason this shouldn't be the same as the return before?

```suggestion
            return ["--verify", "--etherscan-api-key", apiKey];
```